compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -174,6 +174,21 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
     insert_out_quantize(node);                                           \
   }
 
+  // INPUT_NAME is the only activation of NODE
+#define INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(NODE, INPUT_NAME)            \
+  void visit(NODE *node)                                                      \
+  {                                                                           \
+    if (auto input_quant = create_in_quantize(node->INPUT_NAME(), node))      \
+      node->INPUT_NAME(input_quant);                                          \
+                                                                              \
+    auto spilt_out_nodes = loco::succs(node);                                 \
+    for (auto split_out : spilt_out_nodes)                                    \
+    {                                                                         \
+      auto split_out_circle = loco::must_cast<luci::CircleNode *>(split_out); \
```

Comment:
I think it would be better to receive `OUT_NAME` as an argument and use it here.

```suggestion
      auto split_out_circle = loco::must_cast<OUT_NAME *>(split_out); \
```

It is ok to use `luci::CircleNode`, but explicit casting to OUT_NAME would be safer. For example, for split Op,

```
INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT(luci::CircleSplit, input, luci::CircleSplitOut)
```